ApiCalls.js

```diff
@@ -24,4 +24,22 @@ export const addFlyToAPI = async (flyData) => {
   } catch (error) {
     console.error(error);
   }
-};
\ No newline at end of file
+};
+
+export const deleteFly = async (flyId) => {
+    try {
+      const response = await fetch(
+        `https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyId}`,
+        {
+          method: "DELETE",
+          headers: {
+            "Content-Type": "application/json;charset=utf-8",
+          },
+        }
+      );
+      const deletionResponse = await response.json();
```

Comment:
Nice job with this delete setup!  Everything coming together nicely!

Reply:
Thank you! 😄